crates/api-desc/Cargo.toml

```diff
@@ -14,9 +14,9 @@ categories = ["embedded", "no-std", "wasm"]
 [dependencies]
 data-encoding = { version = "2.6.0", default-features = false }
 data-encoding-macro = { version = "0.1.15", default-features = false }
-proc-macro2 = { version = "1.0.83", default-features = false }
-quote = { version = "1.0.36", default-features = false }
-syn = { version = "2.0.66", default-features = false, features = ["printing"] }
+proc-macro2 = { version = "1.0", default-features = false }
+quote = { version = "1.0", default-features = false }
+syn = { version = "2.0", default-features = false, features = ["printing"] }
```

Comment:
Let's use `M.m.p` even if `p` is 0 for consistency. That's what `cargo add` does and it's preserved by `cargo update --breaking`. If we want to drop the patch version (which we probably shouldn't for security reasons), we should do it across the whole project.